src/Featurizers/CountVectorizerFeaturizer.h

```diff
@@ -281,24 +181,36 @@ Details::CountVectorizerEstimatorImpl<MaxNumTrainingItemsV>::CountVectorizerEsti
             }()
         )
     ),
-    _binary(std::move(binary)),
-    _lower(std::move(lower)),
-    _analyzer(std::move(analyzer)),
-    _regex(std::move(regex)),
-    _ngram_min(std::move(ngram_min)),
-    _ngram_max(std::move(ngram_max)) {
+    _tfidfEstimator(
+       TfidfEstimator(
```

Comment:
>TfidfEstimator [](start = 7, length = 14)

This is constructing a temporary TfidfEstimator instances, and then using the move constructor to move from the temporary to _tfidfEstimator. Remove this indirection and pass the constructor parameters directly to _tifdfEstimator.